stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,1781 @@
+                    case .two:
+                        vvlog2(dest.baseAddress!,
+                               src.baseAddress!,
+                               &n)
+                    case .ten:
+                        vvlog10(dest.baseAddress!,
+                                src.baseAddress!,
+                                &n)
+                    }
+                }
+            }
+    }
+    
+    // MARK: Unbiased Exponent
+    
+    /// Calculates the unbiased exponent of each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func unbiasedExponent<U, V>(_ vector: U,
+                                       result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvlogbf(dest.baseAddress!,
+                            src.baseAddress!,
+                            &n)
+                }
+            }
+    }
+    
+    /// Calculates the unbiased exponent of each element in `vector`, writing the result to `result`, double-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func unbiasedExponent<U, V>(_ vector: U,
+                                       result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvlogb(dest.baseAddress!,
+                           src.baseAddress!,
+                           &n)
+                }
+            }
+    }
+}
+
+// Array-Oriented Power Functions
+
+extension vDSP {
+    
+    // MARK: Power
+    
+    /// Calculates each element in `bases` rasied to the power of the corresponding element in `exponents`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter bases: Input base values.
+    /// - Parameter exponents: Input exponents.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func power<T, U, V>(bases: T,
```

Comment:
I think we'd prefer to call this `pow`, but the spelling should match whatever the (about to start) review of basic scalar math functions settles on.

Reply:
Changed to `pow` for now - I can always change again depending on the results of your review.